layer4/assessment.go

```diff
@@ -9,22 +9,22 @@ import (
 	"time"
 )
 
-// TestResult is a struct that contains the results of a single step within a testSet
+// Assessment is a struct that represents an evaluation of a single requirement
 type Assessment struct {
```

Comment:
@eddie-knight Should this type be defined in the cue schema if it communicates the structure of evaluation results?